dev/run-pip-tests

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+FWDIR="$(cd "`dirname $0`"/..; pwd)"
```

Comment:
```
In dev/run-pip-tests line 21:
FWDIR="$(cd "`dirname $0`"/..; pwd)"
         ^-- SC2164: Use cd ... || exit in case cd fails.
             ^-- SC2006: Use $(..) instead of legacy `..`.
                      ^-- SC2086: Double quote to prevent globbing and word splitting.


In dev/run-pip-tests line 22:
cd "$FWDIR"
^-- SC2164: Use cd ... || exit in case cd fails.


In dev/run-pip-tests line 26:
$FWDIR/dev/run-pip-tests-2
^-- SC2086: Double quote to prevent globbing and word splitting.


In dev/run-pip-tests line 31:
  rm -rf `cat ./virtual_env_temp_dir`
         ^-- SC2046: Quote this to prevent word splitting.
         ^-- SC2006: Use $(..) instead of legacy `..`.
```